File: Python/zadanie3.py
print("--- ZADANIE 3: Średnia ocen ucznia ---")

liczba_ocen = int(input("Podaj liczbę ocen: "))
suma_ocen = 0.0

# Pętla for do pobierania kolejnych ocen
for i in range(liczba_ocen):
    ocena = float(input(f"Podaj ocenę {i + 1}: "))
    suma_ocen += ocena

# Obliczanie średniej
srednia = suma_ocen / liczba_ocen

# Wypisanie wyniku (:.2f wymusza wyświetlenie 2 miejsc po przecinku)
print(f"Średnia: {srednia:.2f}")

# Warunek zaliczenia
if srednia >= 3.0:
    print("Uczeń zdał.")
else:
    print("Uczeń nie zdał.")

File: C#/zadanie3.cs
using System;

class Program
{
    static void Main()
    {
        Console.WriteLine("Zadanie 3");

        Console.Write("Podaj liczbę ocen: ");
        int liczbaOcen = Convert.ToInt32(Console.ReadLine());

        double sumaOcen = 0;

        for (int i = 0; i < liczbaOcen; i++)
        {
            Console.Write("Podaj ocenę " + (i + 1) + ": ");
            double ocena = Convert.ToDouble(Console.ReadLine());
            sumaOcen += ocena;
        }

        double srednia = sumaOcen / liczbaOcen;
        Console.WriteLine("Średnia: {0:F2}", srednia);

        if (srednia >= 3.0)
        {
            Console.WriteLine("Uczeń zdał.");
        }
        else
        {
            Console.WriteLine("Uczeń nie zdał.");
        }
    }
}